pkg/providers/aws/provider_redis.go

```diff
@@ -657,15 +665,24 @@ func (p *RedisProvider) exposeRedisMetrics(ctx context.Context, cr *v1alpha1.Red
 	// build generic metrics
 	genericLabels := buildRedisGenericMetricLabels(cr, clusterID, cacheName)
 
+	statusLables := buildRedisStatusMetricsLabels(cr, clusterID, cacheName)
```

Comment:
Just a small typo :) `statusLabels`, think a few other files might have this too

Reply:
:+1: